cypher/cypher-docs/src/docs/dev/execution-plans.asciidoc

```diff
@@ -3,45 +3,69 @@
+
+.Statistics
 Each operator is annotated with statistics.
 
 `Rows`::
-The number of rows that the operator produced. Only available if the query was profiled.
+The number of rows that the operator produced.
+This is only available if the query was profiled.
 `EstimatedRows`::
-If Neo4j used the cost-based compiler you will see the estimated number of rows that will be produced by the operator.
+This is the estimated number of rows -- i.e. the approximate number of rows based on the available statistical information -- that will be produced by the operator.
 The compiler uses this estimate to choose a suitable execution plan.
 `DbHits`::
 Each operator will ask the Neo4j storage engine to do work such as retrieving or updating data.
 A _database hit_ is an abstract unit of this storage engine work.
 The actions triggering a database hit are listed <<execution-plans-dbhits, here>>.
 
 To produce an efficient plan for a query, Neo4j requires information about the database, such as which indexes and constraints are present (see <<cypher-schema>>).
-Statistical information maintained by the database is also used to generate an efficient execution plan; i.e. the query engine uses this information to determine which access patterns will produce the best execution plan. 
+Statistical information maintained by the database is also used to generate an efficient execution plan; i.e. the query engine uses this information to determine which access patterns will produce the best execution plan.
 
 The statistical information maintained by Neo4j is:
 
 . The number of nodes having a certain label.
-. Selectivity per index.
 . The number of relationships by type.
+. Selectivity per index.
 . The number of relationships by type, ending with or starting from a node with a specific label.
 
-More information about how the statistics are kept up to date, as well as configuration options about managing query replanning can be found in the <<operations-manual#statistics-execution-plans, Operations Manual -> Statistics and execution plans>>
+More information about how the statistics are kept up to date, as well as configuration options about managing query replanning and caching can be found in the <<operations-manual#statistics-execution-plans, Operations Manual -> Statistics and execution plans>>
 
-See <<how-do-i-profile-a-query>> for how to view the execution plan for a query.
+<<query-tuning>> describes how to tune Cypher queries.
+In particular, see <<how-do-i-profile-a-query>> for how to view the execution plan for a query and <<query-using>> for how to use planner _hints_ to influence the decisions of the planner when building an execution plan for a query.
 
-For a deeper understanding of how each operator works, refer the relevant section.
+For a deeper understanding of how each operator works, refer to the relevant section in <<execution-plan-operators-summary, this table>>.
```

Comment:
I'm not a fan of links named _this table_ or _here_ or similar. Can we not name it?

Reply:
👍